src/librustdoc/html/static/main.js

```diff
@@ -1234,7 +1240,11 @@
         }
 
         function pathSplitter(path) {
-            return '<span>' + path.replace(/::/g, '::</span><span>');
+            var tmp = '<span>' + path.replace(/::/g, '::</span><span>');
+            if (tmp.endsWith("<span>")) {
+                return tmp.slice(0, tmp.length - 6);
+            }
+            return tmp;
```

Comment:
What about situations where the string doesn't end in `<span>`? Doesn't that still have an unclosed span?

Reply:
No, it always ends with `::`. This is on this string that we make the replacement.